src/caffeine/caffeine.c

```diff
@@ -26,20 +28,19 @@ void caf_caffeinate(mspace* symmetric_heap, intptr_t* symmetric_heap_start, mspa
   GASNET_SAFE(gex_Client_Init(&myclient, &myep, initial_team, "caffeine", NULL, NULL, 0));
```

Comment:
The `gex_TM_t` representing the initial team is an invariant and trivially copyable handle that doesn't change over the lifetime of the job.

Please add a `gex_TM_t` global C variable and store a copy of `initial_team` here, so that we can avoid needlessly passing this handle back and forth across the C/Fortran interface.